elixir/apps/web/lib/web/live_table.ex

```diff
@@ -423,7 +429,11 @@ defmodule Web.LiveTable do
   def reload_live_table!(socket, id) do
     callback = Map.fetch!(socket.assigns.callback_by_table_id, id)
     list_opts = Map.get(socket.assigns[:list_opts_by_table_id] || %{}, id, [])
-    callback.(socket, list_opts)
+
+    case callback.(socket, list_opts) do
+      {:error, _reason} -> push_navigate(socket, to: socket.assigns.uri)
```

Comment:
I'm still trying to get up to speed on all of the changes that went in while I was gone, so I may be overlooking something that would prevent what I'm about to ask, but if there is something other than a one-off race condition that is causing an `:error` to be returned, would this put the browser/page into a redirect loop?

Reply:
It should not because we handle errors differently on page load and table reloads. Table reloads will just give up and reload page, but page load will actually raise an error and handle it properly.

Reply:
Cool, I figured there was something I wasn't quite seeing.  Thanks for the insight. 👍